neofetch

```diff
@@ -8,7 +8,7 @@
 #
 # The MIT License (MIT)
 #
-# Copyright (c) 2015-2021 Dylan Araps
+# Copyright (c) 2015-2020 Dylan Araps
```

Comment:
You're uploading an older version of neofetch. Please pull the latest master branch and make changes there.